scripts/install_services.sh

```diff
@@ -187,7 +187,7 @@ install_Caddyfile() {
     cp /etc/caddy/Caddyfile{,.original}
   fi
   if ! [ -z ${CADDY_PWD} ];then
-  HASHWORD=$(caddy hash-password -plaintext ${CADDY_PWD})
+  HASHWORD=$(caddy hash-password --plaintext "${CADDY_PWD}")
```

Comment:
caddy hash-password -plaintext was causing an error.

Reply:
Thank you for finding this!